src/frontend/js/utils/api/lms/edx-dogwood.ts

```diff
@@ -0,0 +1,39 @@
+import { AuthenticationBackend, LMSBackend } from 'types/commonDataProps';
+import { ApiImplementation } from 'types/api';
+import { handle } from 'utils/errors/handle';
+import EDX from './edx-hawthorn';
+
+/**
+ *
+ * EDX Dogwood API Implementation
+ *
+ * This implementation inherits from Hawthorn implementation.
+ * The `user.me` methods has to be override since `/user/v1/me` route does not
+ * exist in EDX Dogwood & Eucalyptus Rest API.
+ *
+ */
+
+const API = (APIConf: LMSBackend | AuthenticationBackend): ApiImplementation => {
+  return {
+    user: {
+      ...EDX(APIConf).user,
+      me: async () => {
+        return await fetch(`${APIConf.endpoint}/api/mobile/v0.5/my_user_info`, {
```

Comment:
The only difference I see with hawthorn is the url.You can maybe refactor this function and use the url as a parameter ?

Reply:
Yes, good catch, I could refactor this to be able to only overwrite api routes if nothing else has to changed.